program/src/main.rs

```diff
@@ -111,6 +59,7 @@ pub fn main() {
         trusted_light_block,
         target_light_block,
         headers,
+        bridge_commitment,
```

Comment:
As discussed, we'll remove this from the proof inputs

Reply:
removed